src/test/java/pages/CataloguePage.java

```diff
@@ -0,0 +1,83 @@
+package pages;
+
+import elements.Text;
+import indices.Icons;
+import indices.Pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import properties.Property;
+import util.WaitUtils;
+
+
+public class CataloguePage extends BasePage {
+
+    @FindBy(xpath = "//h1[@class='catalog-navigation__title']")
+    protected Text catalogueName;
+
+    protected static final String ICON_OPTION_BUTTON = "//li[@data-id=%s]";
+
```

Comment:
Do you need this space line? and in between other locators as well?

Reply:
removed